client/boot/index.js

```diff
@@ -162,13 +163,14 @@ function boot() {
 
 		// Create layout instance with current user prop
 		Layout = require( 'layout' );
-		layout = ReactDom.render( React.createElement( Layout, {
+
+		layoutComponent = React.createElement( Layout, {
```

Comment:
Really nit-picking at this point, but terminology here may be inaccurate, as this is assigning an element, not a component (similar to but not quite the same as the distinction between a class and instance of the class).

https://facebook.github.io/react/blog/2014/10/14/introducing-react-elements.html

```jsx
layoutElement = React.createElement( /* ... */ );
```